package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "build": "hexo generate",
+    "build": "npm rebuild node-sass && hexo generate",
```

Comment:
Please roll back this change. We need to add `node-sass@4.14` to `package.json` to make this a part of the installation.